src/Sia.Gateway/Hubs/EventsHub.cs

```diff
@@ -14,15 +18,90 @@ namespace Sia.Gateway.Hubs
     public class EventsHub : Hub
     {
         public const string HubPath = "events/live";
-        public EventsHub() : base()
+        private readonly ConcurrentDictionary<string, IFilterByMatch<Event>> _filterLookup;
+        private readonly ILogger<EventsHub> _logger;
+
+        public EventsHub(
+            ConcurrentDictionary<string, IFilterByMatch<Event>> filterLookup,
+            ILoggerFactory loggerFactory
+        ) : base()
         {
+            _filterLookup = filterLookup;
+            _logger = loggerFactory.CreateLogger<EventsHub>();
+        }
 
+        public override async Task OnConnectedAsync()
+        {
+            await base.OnConnectedAsync();
+            _logger.LogInformation(
+                "User {0} connected to EventsHub on connectionId {1}",
+                new object[] { Context.User.Identity.Name, Context.ConnectionId }
+            );
+            await ClearFilter();
+        }
+
+        public override async Task OnDisconnectedAsync(Exception exception)
+        {
+            await ClearFilter();
+            
+            if(exception == null)
```

Comment:
Space between if and condition: 

`if(exception == null)` => `if (exception == null)`